src/kibana/directives/in_time_errors.js

```diff
@@ -0,0 +1,32 @@
+define(function (require) {
+  var module = require('modules').get('kibana');
+  var $ = require('jquery');
+
+  module.directive('inTimeErrors', function () {
+    return {
+      // can't require since there is no controller
+      // require: 'ng-submit',
+      restrict: 'C',
+      link: function ($scope, $elem, attrs) {
```

Comment:
You don't know the name of the form until it's compiled, so you don't know the property to isloate.